### PR TITLE
rewrite change_lib_dependencies.rb

### PR DESCRIPTION
**Pull request by xu-cheng**
* It's fully automated. No manual command is required.
* No need to run with sudo.

### CODE REVIEW

File: other/change_lib_dependencies.rb [hunk 1]
```diff
@@ -1,56 +1,101 @@
...
-  else
-    puts 'Error!'
-    puts stderr
   end
-
 end
 
-puts "Totol #{fix_count}"
\ No newline at end of file
+puts "Totol #{fix_count}"
```
Comment (lhc70000) [inline other/change_lib_dependencies.rb]: Oops, it's my typo (`Total`)

**Comment by lhc70000** [comment]
By the way, next time when starting a new PR (especially on player related features), please raise an issue before so we can discuss in advance, and avoid working on the same feature.

File: other/change_lib_dependencies.rb (hunk 1 above)
Comment (xu-cheng) [inline other/change_lib_dependencies.rb]: Fixed

**Comment by xu-cheng** [comment]
> next time when starting a new PR (especially on player related features), please raise an issue before so we can discuss in advance, and avoid working on the same feature.

Noted.

**Pull request merged**